lib/dynflow/executors.rb

```diff
@@ -8,10 +8,11 @@ module Executors
     # we should wrap it with this method, and we can ensure here to do
     # necessary cleanup, such as cleaning ActiveRecord connections
     def self.run_user_code
-      clear_connections = defined?(::ActiveRecord) && ActiveRecord::Base.connected? && ActiveRecord::Base.connection.open_transactions.zero?
       yield
     ensure
-      ::ActiveRecord::Base.clear_active_connections! if clear_connections
+      if defined?(::ActiveRecord) && ActiveRecord::Base.connected? && ActiveRecord::Base.connection.open_transactions.zero?
```

Comment:
so now, we do not clear the connection if it's still active? It doesn't get cleared sometimes then?

Reply:
I tried to capture what's going on in the commit message of https://github.com/Dynflow/dynflow/pull/352/commits/7bd35862708e0c181988def1d5910249ab4d7ae5